src/Surfnet/ServiceProviderDashboard/Application/CommandHandler/Entity/PublishEntityProductionCommandHandler.php

```diff
@@ -109,26 +105,30 @@ public function handle(PublishEntityProductionCommand $command)
 
         try {
             $this->logger->info(
-                sprintf('Publishing entity "%s" to Manage in production environment', $entity->getNameNl())
+                sprintf('Publishing entity "%s" to Manage in production environment', $entity->getNameEn())
             );
 
             $publishResponse = $this->publishClient->publish($entity);
 
             if (array_key_exists('id', $publishResponse)) {
                 // Send the confirmation mail
                 $this->logger->info(
-                    sprintf('Sending publish request mail to service desk for "%s".', $entity->getNameNl())
-                );
-                $mailCommand = new PublishToProductionMailCommand(
-                    $this->buildMailMessage($entity)
+                    sprintf('Sending publish request mail to service desk for "%s".', $entity->getNameEn())
                 );
-                $this->commandBus->handle($mailCommand);
```

Comment:
Thanks for getting rid of the recursion for handling commands in the commandbus. I thuink this is very valuable on the long run.